### ISSUE TITLE
(URGENT) PDF: Broken draft, docstage and title recognition

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/metanorma-ieee#45]
The current Metanorma IEEE PDF implementation is relying on outdated metadata.

The new metadata format for IEEE encodes title, docstage differently.

## Cover page

1. Cover page title and metadata is incorrect
<img width="629" alt="Screenshot 2022-06-27 at 12 00 08 PM" src="https://user-images.githubusercontent.com/11865/175851907-6b0dce87-cdb8-47eb-bca6-5e59ee0584c9.png">

2. Published cover page should not show because this is a draft. IEEE documents do not use Harmonised Stage Codes but the `:docstage: draft` attribute.

## Title

The title is missing on the first page and the header:
<img width="492" alt="Screenshot 2022-06-27 at 12 01 27 PM" src="https://user-images.githubusercontent.com/11865/175852027-84e6e4b7-931e-45e9-ae78-4007c0918070.png">

IEEE documents do not have a 3-part title, they only have 1 part which is encoded just as `title`.

**Comment by Intelligent2013** [comment]
Currently, xslt 'switches' to the 'draft' template by:
- `<doctype>international-standard</doctype>`
- `version/draft != ''`

But should be:
- `status/stage = 'draft'

**Comment by ronaldtse** [comment]
Right, and also the doctype is "standard" instead of "international-standard". Thanks!

**Comment by ronaldtse** [comment]
<img width="719" alt="Screenshot 2022-06-27 at 8 17 27 PM" src="https://user-images.githubusercontent.com/11865/175929576-84467a03-54ed-4de2-9c69-44041546f76e.png">

So we should go with the Style Manual 😉

**Comment by Intelligent2013** [comment]
IEEE xslt updated:
![image](https://user-images.githubusercontent.com/5559926/175925578-9b165083-decf-4ecf-9f0e-a1ef586f2e19.png)
